chart/flux/README.md

```diff
@@ -225,6 +225,7 @@ The following tables lists the configurable parameters of the Weave Flux chart a
 | `registry.ecr.region`                             | `None`                                               | Restrict ECR scanning to these AWS regions; if empty, only the cluster's region will be scanned
 | `registry.ecr.includeId`                          | `None`                                               | Restrict ECR scanning to these AWS account IDs; if empty, all account IDs that aren't excluded may be scanned
 | `registry.ecr.excludeId`                          | `602401143452`                                       | Do not scan ECR for images in these AWS account IDs; the default is to exclude the EKS system account
+| `registry.ecr.require`                            | `false`                                              | Refuse to start if the AWS API is not available
```

Comment:
This still requires updating

Reply:
It matches the entry in `values.yaml` and the templates, doesn't it?

It does _not_ match the fluxd flag -- the convention in the chart is (evidently) to segregate all the ECR bits into a section.

Reply:
Ahh, true.